audit/projects/k8s-infra-prow-build-trusted/iam.json

```diff
@@ -32,8 +38,7 @@
     },
     {
       "members": [
-        "group:k8s-infra-cluster-admins@kubernetes.io",
-        "serviceAccount:prow-deployer@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
```

Comment:
Expected, this was me manually removing a workaround (workaround: https://github.com/kubernetes/k8s.io/pull/2148#issuecomment-856385138, removal: https://github.com/kubernetes/k8s.io/pull/2190#issuecomment-859077364)